src/store/ChartStore.js

```diff
@@ -552,6 +552,13 @@ class ChartStore {
             params = { periodicity: calculateTimeUnitInterval(granularity) };
         }
 
+        if (params === undefined && symbolObj) {
+            // Remove comparisons before symbol changes
+            for (const field in this.stxx.chart.series) {
+                this.stxx.removeSeries(field);
```

Comment:
@MahboobehMohammadi should we clear `_streams` and `_tickHistoryCache` after removing the series from the chart to prevent the cache from being too fat?

Reply:
@morteza-binary no.its been handled somewhere elese